Add region to sample table

ttung opened the pull request:
### Description
Added a migration that manually writes "North America" to all samples.  I'm not aware of any biohub samples that were from outside of North America, and even if there were, there are certainly no plans to import them into aspen.

Depends on #235 

### Test plan
ran the migration

phoenixAja commented on src/py/aspen/database/models/sample.py:
```diff
@@ -123,6 +123,11 @@ class Sample(idbase, DictMixin):
         },
     )
+    region = Column(
+        String,
```
would it make more sense to make this an enum field instead?

ttung replied:
I looked at the region data for all the gisaid data and it appears very constrained to the continents.  I think it's pretty safe to make it an enum.

phoenixAja commented on src/py/aspen/database/models/accessions.py:
```diff
@@ -51,6 +51,7 @@ def add_to_entity(
         workflow_start_datetime: Optional[datetime.datetime] = None,
         workflow_end_datetime: Optional[datetime.datetime] = None,
     ):
+        """"""
```
rm?

phoenixAja reviewed (approved):
looks good!!

Commit:
Add region to sample table

### Description
Added a migration that manually writes "North America" to all samples.  I'm not aware of any biohub samples that were from outside of North America, and even if there were, there are certainly no plans to import them into aspen.

### Test plan
ran the migration
